web-training/JavaScript/Week 5 - Working with Arrays/Ex1/Ex1.html

```diff
@@ -32,10 +32,61 @@
 			"1": 70
 		};
 		
+		window.onload = () => {
+			loadTable();
+		};
+		
+		function loadTable() {
+			const table = document.getElementById("student-table");
+			const tbody = table.getElementsByTagName("tbody")[0];
+			tbody.innerHTML = "";
+			
+			const selectValue = document.getElementById("sort").value;
+			const classValue = document.getElementById("class").value;
+			
+			const elements = x
+			.map((x, index) => {return {...x, index: index};})
+			.map(person => {
```

Comment:
Perhaps we can extract a function that generates a single row and then ask people to use it in `map`?